index.js

```diff
@@ -0,0 +1,39 @@
+const openMenuBtn = document.querySelector('#open-menu');
+const closeMenuBtn = document.querySelector('#close-menu');
+const mobileNav = document.querySelector('#mobile-nav');
```

Comment:
- [ ] **OPTIONAL**
Great job using the `querySelector` method there, but don't you think it's better if you used the `getElementById` method since you're trying to get an element by its `id` attribute! Please tell me what you think! 😄
_Kindly notice that the `querySelector` is getting the first element it finds in the DOM that having the passed argument. So, if you used the `getElementById` it's going to be `('open-menu')` without the `#`._